src/sgame/sg_buildable.cpp

```diff
@@ -1523,8 +1519,7 @@ static gentity_t *SpawnBuildable( gentity_t *builder, buildable_t buildable, con
 	built->enabled = false;
 	built->spawned = false;
 
-	built->flags = g_indestructibleBuildables.Get() ?
-		FL_GODMODE | FL_NOTARGET : 0;
+	built->flags = 0;
```

Comment:
This line should be reverted, or it will only work for buildables constructed before toggling the cvar.

Reply:
I don't understand?

Toggling the cvar will override whatever value was originally there?

Reply:
Oh I see. I'm dumb

Reply:
Done.